app/helpers/application_helper.rb

```diff
@@ -65,6 +65,17 @@ def pagination_before_after
     APP_CONFIG["pagination"]["before_after"]
   end
 
+  # Returns `active` string if current path includes the passed parameter
+  # or if paremeter is `true
```

Comment:
\`true should be either \`true\` or without the backtick. I'd go without the backtick. Plus, a period is missing at the end.